unitTests/testUtils.hpp

```diff
@@ -102,14 +102,14 @@ inline void forall( INDEX_TYPE const max, LAMBDA && body )
 #endif
 
 // Comparator that compares a std::pair by it's first object.
-template< class A, class B, class COMP=std::less< B > >
+template< class A, class B, class COMP=std::less< A > >
```

Comment:
This isn't really a comparison of the pair, but only the first component of the pair?

Reply:
Yeah `std::pair` has builtin comparators but this is used for the `CRSMatrix` tests where you have pairs of columns and values that need to be sorted by the columns.